db/migrate/20181102173232_add_expired_to_users.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddExpiredToUsers < ActiveRecord::Migration[5.0]
+
+  def change
+    add_column :users, :expired_at, :datetime
```

Comment:
We probably want an index on this column, since we use the scope `unexpired` in the `ldap_authentication` rake tasks